dev-docs/bidders/stellormedia.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: bidder
+title: StellorMedia
+description: Stellor Media Bidder Adapter
+biddercode: stellormedia
+aliasCode: adtelligent
+media_types: video,banner
+tcfeu_supported: true
+gpp_supported: true
```

Comment:
`gpp_supported` is deprecated. Please use `gpp_sids` instead and specify the supported legislations.

See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Reply:
Done